db/migrate/20220118183613_create_groups.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGroups < ActiveRecord::Migration[7.0]
+  def change
+    create_table :groups do |t|
+      t.string :name
```

Comment:
A record without `name`, `from_time` or `to_time` should be considered as valid?
If not there should be a validation at db level as well